business-registry-dashboard/app/pages/magicLink.vue

```diff
@@ -0,0 +1,112 @@
+<script setup lang="ts">
+import {
+  NrRequestActionCodes,
+  FilingTypes
+} from '@bcrs-shared-components/enums'
+
+const { t } = useI18n()
+const brdModal = useBrdModals()
+const route = useRoute()
+const affNav = useAffiliationNavigation()
+const { isAuthenticated } = useKeycloak()
+
+// Define eligible corporation types for incorporation
+const ELIGIBLE_CORP_TYPES = [
+  CorpTypes.BENEFIT_COMPANY,
+  CorpTypes.COOP,
+  CorpTypes.BC_CCC,
+  CorpTypes.BC_COMPANY,
+  CorpTypes.BC_ULC_COMPANY
+]
```

Comment:
Hmm, about Coops...

Coops are LEAR-only, but I wonder if users will get an email for them with a magic link. I don't see any reason why not. Check with Kevin. For this reason, I think they can be left here.

PS In the future, there are 4 continued-in entity types that will be added. But not for Directed Launch.